src/axolotl/utils/mlflow_.py

```diff
@@ -16,3 +16,7 @@ def setup_mlflow_env_vars(cfg: DictDefault):
     # Enable mlflow if experiment name is present
     if cfg.mlflow_experiment_name and len(cfg.mlflow_experiment_name) > 0:
         cfg.use_mlflow = True
+
+    # Enable logging hf artifacts in mlflow if value is truthy
+    if cfg.hf_mlflow_log_artifacts is True:
+        os.environ["HF_MLFLOW_LOG_ARTIFACTS"] = "true"
```

Comment:
The variable wasn't being set in the code above before because it's parsed as a bool- I saw similar code to this elsewhere but if there's a preferable way to set the env var please lmk

Reply:
I assume this means that mlflow prefers to pick up this from the env var rather than being passed the setting somewhere else more explicitly?